components/impls/core/src/main/javascript/cde-core-require-js-cfg.js

```diff
@@ -23,55 +23,57 @@
   var isDebug = typeof document == "undefined" || document.location.href.indexOf("debug=true") > 0;
 
   if(typeof KARMA_RUN !== "undefined") { // unit tests
-    requireCfg.paths['cde/components'] = 'src/main/javascript/amd-components';
+    requireCfg.paths['cde'] = 'src/main/javascript';
```

Comment:
I don't think we should be exposing AMD ids such as `cde/components/Raphael/amd/...`. I realize that the implication is that we would need to define each component's AMD id separately, but I thought, from our conversations that that was already the case.

Reply:
For each custom component there is a mapping to the main "class" that is exposed publicly, internally I restructured according to the outcome of our conversations.